src/hotspot/share/runtime/stackValue.hpp

```diff
@@ -29,6 +29,7 @@
 #include "runtime/handles.hpp"
 
 class BasicLock;
+class frame;
```

Comment:
Seems unrelated to this change. ?/

Reply:
This file was indirectly including frame.hpp via os.hpp. Since I removed os.hpp from this file, I needed to add a declaration for the `frame` class.

Reply:
I don't see any such removal in this file. ??

Reply:
Oh I was mistaken. What I did was removing os.hpp from relocInfo.hpp and safepointMechanismhpp. This exposed the fact that stackValue.hpp was using the `frame` class without a proper declaration.